plutus-benchmark/marlowe/test/data.budgets.golden.tsv

```diff
@@ -1,200 +1,200 @@
-320693635	1461487	13652
-514158343	2015450	13652
-1395583580	6106502	13652
```

Comment:
any idea where these improvements come from?

Reply:
I tweaked the implementations of the existing functions and slightly optimised them.